xclim/ensembles/_robustness.py

```diff
@@ -242,7 +245,36 @@ def change_significance(
         )
         # When p < p_change, the hypothesis of no significant change is rejected.
         changed = pvals < p_change
+    elif test == "brownforsythe-test":
+        if weights is not None:
+            raise NotImplementedError(
+                "'brownforsythe-test' is not currently supported for weighted arrays."
+            )
 
+        p_change = kwargs.setdefault("p_change", 0.05)
+        # Test hypothesis of no significant change
+        # -> Brown-Forsythe test
+        pvals = []
+        for realization in fut.realization.values:
+            fut_real = fut.sel(realization=realization).squeeze()
+            ref_real = ref.sel(realization=realization).squeeze()
+            pvals_ = xr.apply_ufunc(
+                lambda f, r: spstats.levene(f, r, center="median")[1],
+                fut_real,
+                ref_real,
+                input_core_dims=[["time"], ["time"]],
+                output_core_dims=[[]],
+                exclude_dims={"time"},
+                vectorize=True,
+                dask="parallelized",
+                output_dtypes=[float],
+            )
+            pvals_ = pvals_.assign_coords({"realization": realization})
+            pvals_ = pvals_.expand_dims("realization")
+            pvals += [pvals_]
+        pvals = xr.concat(pvals, dim="realization")
```

Comment:
```suggestion
        pvals = xr.apply_ufunc(
            lambda f, r: spstats.levene(f, r, center="median")[1],
            fut,
            ref,
            input_core_dims=[["time"], ["time"]],
            output_core_dims=[[]],
            exclude_dims={"time"},
            vectorize=True,
            dask="parallelized",
            output_dtypes=[float],
        )
```
Wouldn't that work? 
From xarray's doc of `apply_ufunc`:

> vectorize ([bool](https://docs.python.org/3/library/functions.html#bool), optional) – If True, then assume func only takes arrays defined over core dimensions as input and vectorize it automatically

Reply:
@aulemahal: perfect, this works! Then we can change `input_core_dims` and `output_core_dims` for all methods  ('manwhitney-utest', 'ttest' and 'welch-ttest') and thus we can get rid of `realization` in the `apply_ufunc` calls. I'll push that.

Reply:
Now "xclim/testing/tests/test_ensembles.py::test_change_significance" fails. What exactly is the difference between:
```
input_core_dims=[[realization, "time"], [realization, "time"]]
output_core_dims=[[realization]]
```
and
```
input_core_dims=[["time"], ["time"]]
output_core_dims=[[]]
```
Can we apply this change to all "methods"? If no why with "brownforsythe-test".
Sorry for so many xarray-related questions :/

Reply:
I'm not sure if I understand. @aulemahal can you follow-up on this?